core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
I mean we might have both methods `should` and `shouldNot` defined on `RequestReader` so we can name all the rules as `BeX` like `BePositive`. It seems nice to me:

```scala
RequiredParam("a") shouldNot BePositive
```